commands/project_init.go

```diff
@@ -0,0 +1,76 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/fatih/color"
+	"github.com/platformsh/platformify/commands"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	ProjectInitCmd.SetHelpFunc(projectInitHelpFn)
+	RootCmd.AddCommand(ProjectInitCmd)
+}
+
+var ProjectInitCmd = &cobra.Command{
+	Use:           "project:init",
+	Short:         "Initialize the needed YAML files for your Platform.sh project",
```

Comment:
```suggestion
	Short:         "Creates Platform.sh-related starter YAML files for your project",
```

What do you think about the wording @lolautruche?

Reply:
I applied the suggestion for now.
@lolautruche if you want to make improvements into the text, let me know, I will fix it

Reply:
@lolautruche should I change the text above? I think we could leave the text as is and create a task to discuss the vendorization, because this is not the only place in the code where we mention P.sh

Reply:
ok, thank you!
[new issue](https://github.com/platformsh/cli/issues/72)